format/spec.md

```diff
@@ -301,7 +301,7 @@ Tables are configured with a **partition spec** that defines how to produce a tu
 *   A **transform** that is applied to the source column(s) to produce a partition value
 *   A **partition name**
 
-The source column, selected by id, must be a primitive type and cannot be contained in a map or list, but may be nested in a struct. For details on how to serialize a partition spec to JSON, see Appendix C.
+The source column(s), selected by id(s), must be a primitive type and cannot be contained in a map or list, but may be nested in a struct. The ability to have multiple source columns was added in V3, whereas previously only a single column was allowed. For details on how to serialize a partition spec to JSON, see Appendix C.
```

Comment:
* I don't think that using "(s)" is helpful. It's distracting and I think it is just as clear to say "The source columns must be primitive types".
* The new statement is a bit too strong since we are adding multi-column transforms in v2. Clients must be able to interpret Iceberg metadata with multiple columns in order to support v3, but can optionally support multiple source columns in v2.

Reply:
Fixed thanks.  Initially was thinking to also support this in V2 with a flag to make it behave like V3.  But you are right we should call it out explicitly in the spec.